include/lemlib/chassis/differential.hpp

```diff
@@ -198,8 +200,8 @@ class Differential : public Chassis {
          * @param angularSettings settings for the angular controller
          * @param sensors sensors to be used for odometry
          */
-        Differential(Drivetrain drivetrain, ControllerSettings linearSettings, ControllerSettings angularSettings,
-                     OdomSensors sensors);
+        Differential(Drivetrain drivetrain, ControllerSettings<Length> lateralSettings,
```

Comment:
```suggestion
        Differential(Drivetrain drivetrain, ControllerSettings<Length> linearSettings,
```
Change lateral to linear